src/frontend/context/ObservationsContext.tsx

```diff
@@ -1,60 +1,74 @@
-// @flow
 import * as React from "react";
 
 import api from "../api";
 import type { LocationContextType } from "./LocationContext";
 import type { Status } from "../types";
+import { Position } from "mapeo-schema";
 
-export type ObservationAttachment = {
-  id: string,
-  type?: string,
-};
+export interface ObservationAttachment {
+  id: string;
+  type?: string;
+}
 
-export type ObservationValue = {
-  lat?: number | null,
-  lon?: number | null,
-  deviceId?: string,
+export interface ObservationValue {
+  lat?: number | null;
+  lon?: number | null;
+  deviceId?: string;
   metadata?: {
-    location?: LocationContextType | void,
-    manualLocation?: boolean,
-  },
-  refs?: Array<{ id: string }>,
-  attachments?: Array<ObservationAttachment>,
-  tags: { [string]: any },
-};
+    location?: LocationContextType | void;
+    manualLocation?: boolean;
+  };
+  refs?: Array<{ id: string }>;
+  attachments?: Array<ObservationAttachment>;
+  tags: { [key: string]: any };
+}
 
-export type Observation = {
-  id: string,
-  version: string,
-  created_at: string,
-  timestamp?: string,
-  userId?: string,
-  type: "observation",
-  links?: string[],
-  schemaVersion: 4,
-  value: ObservationValue,
-};
+export interface Observation {
+  id: string;
+  version: string;
+  created_at: string;
+  timestamp?: string;
+  userId?: string;
+  type: "observation";
+  links?: string[];
+  schemaVersion: 4;
+  value: ObservationValue;
+  metadata?: {
+    lastSavedPosition?: Position;
+    manualLocation?: boolean;
+    position?: Position;
+    positionProvider?: {
+      gpsAvailable?: boolean;
+      locationServicesEnabled?: boolean;
+      networkAvailable?: boolean;
+      passiveAvailable?: boolean;
+    };
+  };
+}
```

Comment:
hm not sure. looks like it was added via https://github.com/digidem/mapeo-mobile/pull/723/commits/9b52009da9ebeb6325812f41b45a953f6c6eed91 (probably because TS was complaining about a missing `metadata` field or something)

Reply:
So apparantly I did this....but I really dont remember doing this, but Im looking more into it now

Reply:
Oh yeah! I see what is happening. The API call is attempting to map `ServerObservation` onto on object of type `Observation`:
<img width="582" alt="image" src="https://user-images.githubusercontent.com/67773827/132263035-42651b93-ba89-4521-b30e-801815c97ab7.png">

Reply:
So should convert from server actually be returning 2 objects, `Observation` and `Observation Value`?